dlt/destinations/queries.py

```diff
@@ -121,3 +165,259 @@ def build_select_expr(
     select_expr = sge.Select(expressions=columns_expr).from_(table_expr)
 
     return select_expr
+
+
+def build_insert_expr(
+    table_name: str,
+    columns: List[str],
+    quoted_identifiers: bool = True,
+) -> sge.Insert:
+    """Builds a SQL Insert expression with placeholders."""
+    table_expr = sge.Table(this=sge.to_identifier(table_name, quoted=quoted_identifiers))
+
+    columns_expr = [sge.to_identifier(col, quoted=quoted_identifiers) for col in columns]
+
+    placeholders_expr = sge.Values(
+        expressions=[sge.Tuple(expressions=[sge.Placeholder() for _ in enumerate(columns)])]
```

Comment:
Maybe it is possible to set `Literal.string('%s')` instead of `sge.Placeholder()` here and skip the replacing of placeholders via regex? Did you try that?